source/application-configuration.rst

```diff
@@ -4,44 +4,86 @@ Application Configuration
 Properties based configuration
 ==============================
 Linux for Health application settings are configured in an application.properties file stored within the project under *src/main/resources*.
-Linux for Health settings are stored within the following namespaces:
+Linux for Health properties are reside within the following namespaces:
 
-- linuxforhealth.connect.component
-- linuxforhealth.connect.endpoint
+- lfh.connect.bean
```

Comment:
I'm tripping a bit over the bean concept.  Everyone has likely heard of Java beans, but few have probably worked with them or written them.  However, a class is a more familiar construct and a Java bean is just a special type of class.  You do mention Bean/POJO further below, but POJO isn't likely familiar to everyone either.  If we could add a couple of sentences that describe why someone might need to use lfh.connect.bean when adding a route, that could help.  You do have the hl7 example, so perhaps a couple of sentences following that to describe why a developer might need to use bean properties could make these first 2 sections more intuitive.

Reply:
Sure I'll add additional context for the "bean". I went with "bean" rather than component, class, etc since that's what the underlying Camel framework is using
```
    /**
     * Binds the given <code>name</code> to the <code>bean</code> object, so
     * that it can be looked up inside the CamelContext this command line tool
     * runs with.
     *
     * @param name the used name through which we do bind
     * @param bean the object to bind
     */
    public void bind(String name, Object bean) {
        registry.bind(name, bean);
    }
```
Any Java developer should be familiar with POJO, beans, etc as all of the popular frameworks utilize this concept. I'll add brief explanation and linkage for a "bean" and then will explain how those are utilized within the Camel framework.